src/sb_geom_msgs/msg/Polygon2D.msg

```diff
@@ -0,0 +1,3 @@
+#A specification of a polygon where the first and last points are
```

Comment:
This is the same specification as geometry_msgs::Polygon, maybe it make it more specific by saying, "A specification of a 2-dimensional polygon where..."

Reply:
changed as per suggestion